api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -69,7 +69,7 @@
     userService.logAdminUserAction(
         user.getUserId(),
         "user access to  " + groupName + " domain",
-        (previousAccess != null) ? previousAccess.toString() : "null",
+        previousAccess,
         DataAccessLevel.REVOKED.toString());
```

Comment:
`DataAccessLevel.REVOKED.toString()` is a bit redundant since you call `toString` on the argument. `DataAccessLevel.REVOKED` would also work.